msteams-platform/concepts/design/plan-responsive-tabs-for-teams-mobile.md

```diff
@@ -69,7 +69,7 @@ In the second stage, the users can work on the existing charts and visuals from
 * Filter data.
 * Create bookmarks.
 
-:::image type="content" source="../../assets/images/app-fundamentals/dashboarding-and-data-visualization-apps-quick-actions.png" alt-text="Quick actions on the existing chart and visuals.":::
+:::image type="content" source="../../assets/images/app-fundamentals/dashboarding-and-data-visualization-apps-quick-actions.png" alt-text="The screenshot describes the quick actions on the existing chart and visuals.":::
```

Comment:
Same as above. This review is getting tiresome now :( 

CC @v-habibaumme

Reply:
changes the screenshot to image